core/aws_ddk_core/stages/databrew_transform.py

```diff
@@ -0,0 +1,125 @@
+        environment_id : str
+            Identifier of the environment
+        job_name : Optional[str]
+            The name of a preexisting Databrew job to run. If None, a Databrew job is created
+        job_role : Optional[IRole]
+            The job execution role
+        job_type : Optional[str]
+            The type of job to run. If None, the job type is inferred if the recipe is provided or not.
+        dataset_name : Optional[str]
+            The name of the dataset to use for the job.
+        recipe_name : Optional[CfnJob.RecipeProperty]
+            The name of the recipe to use for the job.
+        outputs : Optional[List[CfnJob.OutputProperty]]
+            The output properties for the job.
+        state_machine_input : Optional[Dict[str, Any]]
+            The input dict to the state machine
+        additional_role_policy_statements : Optional[List[PolicyStatement]]
+            Additional IAM policy statements to add to the state machine role
+        state_machine_failed_executions_alarm_threshold: Optional[int]
+            The number of failed state machine executions before triggering CW alarm. Defaults to `1`
+        state_machine_failed_executions_alarm_evaluation_periods: Optional[int]
+            The number of periods over which data is compared to the specified threshold. Defaults to `1`
+        """
+        super().__init__(scope, id)
+
+        self._event_detail_type: str = f"{id}-event-type"
+
+        # If None, a Databrew job is created
+        self._job: Optional[CfnJob] = None
+        if not job_name:
+            self._job = DatabrewFactory.job(
+                self,
+                id=f"{id}-job",
+                environment_id=environment_id,
+                role=job_role,
+                type=job_type,
+                dataset_name=dataset_name,
+                recipe=recipe,
+                outputs=outputs,
+            )
+            job_name = self._job.job_name
+
+        # Create GlueDataBrewStartJobRun step function task
+        start_job_run: GlueDataBrewStartJobRun = GlueDataBrewStartJobRun(
+            self,
+            "start-job-run",
+            name=job_name,
+            integration_pattern=IntegrationPattern.RUN_JOB,
+        )
+
+        # Build state machine
+        self.build_state_machine(
+            id=f"{id}-state-machine",
+            environment_id=environment_id,
+            definition=(start_job_run.next(Succeed(self, "success"))),
+            state_machine_input=state_machine_input,
+            additional_role_policy_statements=additional_role_policy_statements,
+            state_machine_failed_executions_alarm_threshold=state_machine_failed_executions_alarm_threshold,
+            state_machine_failed_executions_alarm_evaluation_periods=state_machine_failed_executions_alarm_evaluation_periods,  # noqa
+        )
+
+    @property
+    def job(self) -> Optional[CfnJob]:
+        """
+        Return: Optional[CfnJob]
+            The Glue job
```

Comment:
Minor - copy-paste

Reply:
Oops !!
Updated `The Glue Job` to `The Databrew Job`

Reply:
Resolving the conversation for now

Reply:
Thanks